test/02/t0262a.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# srecord - Manipulate EPROM load files
+# Copyright (C) 2014 Markus Heidelberg
```

Comment:
I assume the copyright line is just copy'n'paste from another file

Reply:
My mistake, did not update copyright fields when creating each test.